tools/BuildAssets/psModules/CodeGenerationModules/AutoRestCodeGenerationModule/AutoRestCodeGenerationModule.psm1

```diff
@@ -86,7 +86,8 @@ function launchProcess {
     $p.WaitForExit()
     $stdout = $p.StandardOutput.ReadToEnd()
     $stderr = $p.StandardError.ReadToEnd()
-    Write-InfoLog $stdout 
+    # send std output to the console only
```

Comment:
@dsgouda can we add adequate comments to explain the design or whatever that will help in understanding why we made certain design decisions. Trust me you will forget your own code in few months, so let's add some comments.
This will also help in external contributors to contribute to this code base.

Reply:
Agreed